prefetch.bs

```diff
@@ -151,6 +151,19 @@ The user agent may remove elements from the [=prefetch buffer=] even if they are
     1. Return |navigationParams|.
 </div>
 
+A <dfn export>prefetch IP anonymization policy</dfn> is either null or a [=tuple=] consisting of the string "`anonymous-client-ip-when-cross-origin`" and an [=origin=].
```

Comment:
This seems like some attempt to emulate a tagged union? Could we instead do something like:

- A prefetch anonymization policy is either null or an IP anonymization policy.
- An IP anonymization policy has an origin, which is an origin.

Then say things like "If policy is an IP anonymization policy, let origin be policy's origin".